tests/governance_history.py

```diff
@@ -61,7 +61,11 @@ def run(args):
         network.start_and_join(args)
         primary, _ = network.find_primary()
 
-        ledger_filename = network.find_primary()[0].remote.ledger_path()
+        ledger_directory = network.find_primary()[0].remote.ledger_path()
+        # For now, this test only works with one ledger file
+        for l in os.listdir(ledger_directory):
```

Comment:
Hack for now. Our `ledger.py` infra should be able to work on multiple chunks. I'll ticket that.

Reply:
https://github.com/microsoft/CCF/issues/1275